cmd/syncthing/monitor.go

```diff
@@ -181,6 +186,43 @@ func copyStderr(stderr io.ReadCloser, dst io.Writer) {
 					panicFd.WriteString(line)
 				}
 				stdoutMut.Unlock()
+
+				if optPanicReporting {
+					// Prepare query for collecting error log
+					type Report struct {
+						Error       string
+						Version     string
+						LongVersion string
+						Os          string
+						Arch        string
+					}
+					report := &Report{Error: line[:len(line)-1],
+						Version:     Version,
+						LongVersion: LongVersion,
+						Os:          runtime.GOOS,
+						Arch:        runtime.GOARCH}
+					b, err := json.Marshal(report)
+					if err == nil {
+						var jsonString = `docs=` + string(b)
+						// Send error log
+						resp, err := http.Post("http://188.166.50.65:27080/logsdb/logs/_insert", "application/x-www-form-urlencoded", strings.NewReader(jsonString))
```

Comment:
Server side I used: mongo + sleepy.mongoose mentioned above + mongo-express for viewing database. But I think it would be a good idea to set up Sentry under data.syncthing.net/panic and send this JSON blob of type application/json there.